### PR TITLE
fix(journey-maps): automatically set the correct floor level for journey transfers

### PR DESCRIPTION
**Pull request by swiss-chris**
Refs: beta.sbb.ch

### CODE REVIEW

File: src/journey-maps/angular/services/map/map-leit-poi-service.ts [hunk 1]
```diff
@@ -51,18 +51,19 @@ export class SbbMapLeitPoiService {
...
-        );
-        routeStartLevel = routeStartLevelFeature
-          ? Number(routeStartLevelFeature.properties!.routeStartLevel)
-          : SbbMapLeitPoiService._defaultLevel;
-      }
+      const routeStartLevelFeature = featureCollection.features.find(
+        (f) =>
+          !!f.properties?.step &&
+          f.properties?.routeStartLevel &&
+          (!legId || f.properties?.legId === legId),
```
[the remark this answers is not in the document]
Comment (swiss-chris) [reply src/journey-maps/angular/services/map/map-leit-poi-service.ts:58]: We could have multiple features with different `routeStartLevel ` inside the same `/journey`.

**Review by davidkuehner — APPROVED** [review]
lgtm

**Comment by mhaertwig** [comment]
Isn't it a fix?

**Comment by swiss-chris** [comment]
> Isn't it a fix?

Indeed! I noticed it myself this time and "fixed" it :)

**Review by mhaertwig — APPROVED** [review]

**Pull request merged**